src/System.Net.Mail/ref/System.Net.Mail.cs

```diff
@@ -158,12 +158,6 @@ public enum MailPriority
         Normal = 0,
     }
     public delegate void SendCompletedEventHandler(object sender, System.ComponentModel.AsyncCompletedEventArgs e);
-    public enum SmtpAccess
-    {
-        Connect = 1,
-        ConnectToUnrestrictedPort = 2,
-        None = 0,
-    }
```

Comment:
I thought about keeping it but decided to go ahead and just remove it from here and .NET Standard since it is only ever used by the CAS APIs.